RefRed/reduction/live_reduction_handler.py

```diff
@@ -95,6 +106,42 @@ def run(self):
 
         StatusMessageHandler(parent=self.parent, message='Done!', is_threaded=True)
 
+    def export(self):
+        """
+        Export a basic script that will reduce every run
+        """
+        # Ask for the output file path
+        run_number = self.parent.big_table_data[0, 0].run_number
+        default_filename = os.path.join(self.parent.path_ascii, "REFL_%s_data_reduction_script.py" % run_number)
+        filename, _ = QFileDialog.getSaveFileName(self.parent, "Python script", default_filename)
+
+        # If the user hits the cancel button, just exit
+        if not filename:
+            return
+
+        # Gather information and generate script
+        o_general_settings = GlobalReductionSettingsHandler(parent=self.parent)
+        common_pars = o_general_settings.to_dict()
+
+        script = "# Reduction script generated by refred on %s\n" % time.ctime()
+        script += "import json\n"
+        script += "from lr_reduction import template\n"
```

Comment:
is `lr_reduction` the source package for _LiquidsReflectometer_?

Reply:
Yes.